k8s/environments/bangladesh-staging/configmaps/simple-server.yaml

```diff
@@ -44,6 +44,8 @@ data:
   SIMPLE_SERVER_ENV: "demo"
   SIMPLE_SERVER_HOST: "api-demo.bd.simple.org"
   SIMPLE_SERVER_HOST_PROTOCOL: "https"
+  STALE_EXPERIMENT_MIN_DAYS_SINCE_LAST_VISIT: "35"
+  STALE_EXPERIMENT_MAX_DAYS_SINCE_LAST_VISIT: "365"
```

Comment:
[nit, nonblocking] I am not sure what bangladesh-staging is, assuming this is the demo env. Can we make this the same config as production (ie, 35-90) so that they behave similarly?

Reply:
Oh yeah, good point. Yes, it's the demo env

Reply:
🔥